pygmt/base_plotting.py

```diff
@@ -606,47 +606,53 @@ def grdview(self, grid, **kwargs):
 
     @fmt_docstring
     @use_alias(
-        R="region",
-        J="projection",
+        A="straight_lines",
         B="frame",
-        S="style",
+        C="cmap",
+        D="offset",
+        E="error_bars",
+        F="connection",
         G="color",
+        I="intensity",
+        J="projection",
+        L="close",
         N="no_clip",
-        W="pen",
-        i="columns",
-        l="label",
-        C="cmap",
+        R="region",
+        S="style",
         U="timestamp",
         V="verbose",
+        W="pen",
         X="xshift",
         Y="yshift",
+        Z="zvalue",
+        i="columns",
+        l="label",
         p="perspective",
         t="transparency",
     )
     @kwargs_to_strings(R="sequence", i="sequence_comma", p="sequence")
     def plot(self, x=None, y=None, data=None, sizes=None, direction=None, **kwargs):
         """
-        Plot lines, polygons, and symbols on maps.
-
-        Used to be psxy.
+        Plot lines, polygons, and symbols in 2-D.
 
         Takes a matrix, (x,y) pairs, or a file name as input and plots lines,
         polygons, or symbols at those locations on a map.
 
         Must provide either *data* or *x* and *y*.
 
-        If providing data through *x* and *y*, *color* (G) can be a 1d array
-        that will be mapped to a colormap.
+        If providing data through *x* and *y*, *color* can be a 1d array that
+        will be mapped to a colormap.
 
-        If a symbol is selected and no symbol size given, then psxy will
+        If a symbol is selected and no symbol size given, then plot will
         interpret the third column of the input data as symbol size. Symbols
         whose size is <= 0 are skipped. If no symbols are specified then the
-        symbol code (see *S* below) must be present as last column in the
-        input. If *S* is not used, a line connecting the data points will be
-        drawn instead. To explicitly close polygons, use *L*. Select a fill
-        with *G*. If *G* is set, *W* will control whether the polygon outline
-        is drawn or not. If a symbol is selected, *G* and *W* determines the
-        fill and outline/no outline, respectively.
+        symbol code (see **symbol** below) must be present as last column in
```

Comment:
It's *style*, not *symbol*.
```suggestion
        symbol code (see **symbol** below) must be present as last column in
```

Reply:
Good catch!